x/amm/types/constants.go

```diff
@@ -23,14 +24,14 @@ var (
 	// This is done so that smooth weight changes have enough precision to actually be smooth.
 	GuaranteedWeightPrecision int64 = 1 << 30
 
-	oneHalf           = math.LegacyMustNewDecFromStr("0.5")
-	twoDec            = math.LegacyMustNewDecFromStr("2")
-	ln2               = math.LegacyMustNewDecFromStr("0.693147180559945309")
-	inverseLn2        = math.LegacyMustNewDecFromStr("1.442695040888963407")
-	euler             = math.LegacyMustNewDecFromStr("2.718281828459045235")
+	oneHalf           = osmomath.MustNewBigDecFromStr("0.5")
+	twoDec            = osmomath.MustNewBigDecFromStr("2")
+	ln2               = osmomath.MustNewBigDecFromStr("0.693147180559945309")
+	inverseLn2        = osmomath.MustNewBigDecFromStr("1.442695040888963407")
+	euler             = osmomath.MustNewBigDecFromStr("2.718281828459045235")
```

Comment:
ln 2 to "0.693147180559945309417232121458176568"
inverseLn2 to "1.442695040888963407359924681001892137"
euler to "2.718281828459045235360287471352662497"